pkg/master/thirdparty/thirdparty.go

```diff
@@ -60,8 +64,8 @@ type ThirdPartyResourceServer struct {
 
 	// storage for third party objects
 	thirdPartyStorageConfig *storagebackend.Config
-	// map from api path to a tuple of (storage for the objects, APIGroup)
-	thirdPartyResources map[string]*thirdPartyEntry
+	// map from api path to a tuple of (versioned storage for the objects, APIGroup)
+	thirdPartyResources map[string]*thirdPartyGroup
```

Comment:
keying by path is weird.  Why not key by group name and have the tprGroup hold a path value?